simp_le.py

```diff
@@ -1297,13 +1311,14 @@ def persist_new_data(args, existing_data):
         response, validation = challb.response_and_validation(client.key)
         save_validation(roots[name], challb, validation)
 
-        verified = response.simple_verify(
-            challb.chall, name, client.key.public_key())
-        if not verified:
-            logger.warning('%s was not successfully self-verified. '
-                           'CA is likely to fail as well!', name)
-        else:
-            logger.info('%s was successfully self-verified', name)
+        if args.self_verify:
+            verified = response.simple_verify(
+                challb.chall, name, client.key.public_key())
+            if not verified:
+                logger.warning('%s was not successfully self-verified. '
+                               'CA is likely to fail as well!', name)
```

Comment:
If we're doing this because self-verification is unrelaible, we should probably tone-down the message here; the CA probably won't fail.